### ISSUE TITLE
Create "render current document"

### ISSUE THREAD

**Issue by KINGSABRI** [John-K/inkdrop-markmap#1]
Hi @John-K 
Thank you for making the plugin available. 
I want to ask you if you can create a context menu to render the currently open or selected document without the need to be put in the tag (```markmap)

The way I imagine it is when I open a document, I right-click then select "render to markmap", a new pop-up window opens with the mindmap for the opened document. In another word, it's a temporary pop-up window that renders the currently opened markdown file within inkdrop.

Thank you

**Comment by Dracoir** [comment]
I agree with @KINGSABRI.  Kind of pointless if I can't export the SVG image and share with others.

The markmap library seems to support this, with examples: https://markmap.js.org/repl

Please advise.

**Comment by John-K** [comment]
The plugin is currently only invoked by the ` ```markmap ` code block, I'm not sure how I could easily modify the plugin to take the entire document and treat as markmap code.

**Comment by John-K** [comment]
I think I've found a way to do this.
What functionality would you like to see in this?
Would you just want to be able to save the SVG file?

**Comment by John-K** [comment]
Apparently saving the SVG doesn't always work well with markmap - even from their demo page.

Would saving a PNG be acceptable?

**Comment by KINGSABRI** [comment]
PNG would do the job
